File: PRAK505_2310817210022_MUHAMMAD_ADH-DHIYA_US_SALIM.c
#include <stdio.h>

void Biodata(int TahunLahir, char NamaLengkap[], char AsalKota[])
{
    int TahunSekarang = 2023;
    int umur = TahunSekarang - TahunLahir;
    printf("\nPerkenalkan Nama Saya : %s\n", NamaLengkap);
    printf("Umur Saya : %d\n", umur);
    printf("Saya adalah angkatan : %d\n", TahunSekarang);
    printf("Asal Saya dari : %s\n", AsalKota);
}
int main()
{
    int TahunLahir;
    char NamaLengkap[100], AsalKota[100];
    scanf(" %d",&TahunLahir);
    scanf(" %[^\n]%*c", NamaLengkap);
    scanf(" %[^\n]%*c", AsalKota);
    Biodata(TahunLahir, NamaLengkap, AsalKota);

    return 0;
}

File: PRAK505_2310817210022_MUHAMMAD_ADH-DHIYA_US_SALIM.py
def biodata(tahun_lahir, nama_lengkap, asal_kota):
    tahun_sekarang = 2023
    umur = tahun_sekarang - tahun_lahir
    print(f"\nPerkenalkan Nama Saya : {nama_lengkap}")
    print(f"Umur Saya : {umur}")
    print(f"Saya adalah angkatan : {tahun_sekarang}")
    print(f"Asal Saya dari : {asal_kota}")

def main():
    tahun_lahir = int(input())
    nama_lengkap = input().strip()
    asal_kota = input().strip()

    biodata(tahun_lahir, nama_lengkap, asal_kota)

if __name__ == "__main__":
    main()
